bin/ghe-backup

```diff
@@ -69,7 +69,7 @@ if [ -f ../in-progress ]; then
   progress=$(cat ../in-progress)
   snapshot=$(echo "$progress" | cut -d ' ' -f 1)
   pid=$(echo "$progress" | cut -d ' ' -f 2)
-  if ! ps -p $pid -o command= | grep ghe-backup; then
+  if ! ps -p $pid -o command= 2>/dev/null | grep ghe-backup; then
```

Comment:
It'd be good if we could still provide protection against multiple simultaneous backups to those who lack `-o` support.

As an alternative, could we drop the use of `-o` altogether?

```sh
if ! ps -p $pid | grep ghe-backup; then
```